tethys_gizmos/static/tethys_gizmos/js/cesium_map_view.js

```diff
@@ -274,10 +274,18 @@ var CESIUM_MAP_VIEW = (function() {
         }
 
         for (let model of m_models) {
-            // options is where the model data is store.
-            let model_option = cesium_options(model.options);
+            var model_option;
+            // if model has options attribute, it should come from MVLayer.
+            if (model.options) {
+                // options is where the model data is store.
```

Comment:
// The options attribute is where the Cesium model definition is stored in MVLayer objects.